server_admin/topics/clients/oidc/con-advanced-settings.adoc

```diff
@@ -91,11 +91,20 @@ The procedure to select the algorithm is:
 . Open *Fine Grain OpenID Connect Configuration*.
 . Select the algorithm from *ID Token Encryption Content Encryption Algorithm* pulldown menu.
 
-[[_mapping-acr-to-loa]]
+[[_mapping-acr-to-loa-client]]
 *ACR to Level of Authentication (LoA) Mapping*
 
-In the advanced settings of a client, you can define which `Authentication Context Class Reference (ACR)` value is mapped to which `Level of Authentication (LoA)`. The ACR can be any value, whereas the LoA must be numeric.
-The acr claim in also included in the access token and ID token.
+In the advanced settings of a client, you can define which `Authentication Context Class Reference (ACR)` value is mapped to which `Level of Authentication (LoA)`.
+This mapping can be specified also at the realm as mentioned in the <<_mapping-acr-to-loa-realm,ACR to LoA Mapping>>. It is recommended to configure this at the
+realm level, which allows to share the same settings across multiple clients.
+
+The `Default ACR Values` can be used to specify the default values when the login request is sent from this client to {project_name} without `acr_values` parameter and without
```

Comment:
It might be good that L101 and L102 be one sentence.

Reply:
Same as above

Reply:
Replied in the comment https://github.com/keycloak/keycloak-documentation/pull/1428#discussion_r824468705 . We can keep the conversation there related to line feeds.

Reply:
Yes, I agree.